### ISSUE TITLE
Retire "Sieve of Eratosthenes"

### ISSUE THREAD

**Issue by ejini6969** [codewars/content-issues#105]
[Sieve of Eratosthenes](https://www.codewars.com/kata/55f0b69fe3ef582c4100008a)

The kata is basically just a dup of finding prime numbers within specified range , with a lot of issues that allow inefficient solution to pass in a breeze Currently only in Haskell and Java. Basically the same case with #95 which is impossible to make such enforcements to happen!

**Comment by hobovsky** [comment]
I agree that the focus on a solution rather than problem is not a good thing. I also believe there are other kata of similar difficulty which ask for primes up to some bound?  
What i like tho is that this kata aims at some educational value by describing the algorithm (i did not dwell too deep into how well). When the kata gets retired, are we ok with losing the description of the concept?

**Comment by ejini6969** [comment]
> When the kata gets retired, are we ok with losing the description of the concept?

IMO it's perfectly fine considering the fact that the entire description introduction about sieve is taken from Wikipedia ( same goes to that animated image) If the solver wants to learn, they can just google it! There's also a lot of other katas out there that  provide such solutions (some even with detailed explanations) which link to corresponding pages ~~

**Comment by akar-0** [comment]
I vote retirement. The intention and initial idea are good, but nothing is enforced nor enforceable.

**Comment by benjaminzwhite** [comment]
Vote retirement also. 

Late night thought before sleeping though: one (interesting? novel?) way to do this if you want an "enforceable" Sieve of Eratosthenes kata in the database could be to ask users to return, not the primes, but the **sieved numbers** in the order in which they were sieved.

So for input `n = 100`, return `[4, 6, 8, 10, 12, ...,100, 9, 15, 21, 27, ..., 99, 25, ...]`

Or, simpler, for an input `n`, what is the **last** element of the above list (for `n = 100` it is `91`).

**Comment by Blind4Basics** [comment]
note: I found one single other kata talking explicitely about sieve, and it's a custom sieve. So "even if" nothing is effectively enforced/enforcable, _maybe_ it could be worth keeping? Or is there another kata about normal sieve that could use a better name?

note: I'm not _against_ a retirement either... Just considering options.

**Comment by ejini6969** [comment]
> note: I found one single other kata talking explicitely about sieve, and it's a custom sieve. 

Can you provide a link to the kata here?
